### PR TITLE
Update postgres.rb

### PR DESCRIPTION
**Pull request by kidbrax**
adding a check for /var/lib/postgresql/data which is used in their docker images

### CODE REVIEW

**Comment by kidbrax** [comment]
Signed-off-by: Braxton Beyer <braxton@braxtonbeyer.com>

File: lib/resources/postgres.rb [hunk 1]
```diff
@@ -36,6 +36,9 @@ def initialize
         version = if inspec.directory('/var/lib/pgsql/data').exist?
                     warn 'Found /var/lib/pgsql/data. Assuming postgresql install uses un-versioned directories.'
                     nil
+                  elsif inspec.directory('/var/lib/postgresql/data').exist?
```
Comment (chris-rock) [inline lib/resources/postgres.rb:39]: Just to be sure, is that docker specific or os specific?

**Review by chris-rock — COMMENTED** [review]
Thank you for your contribution @kidbrax  Can you share a little bit more feedback? Which operating system is involved?

File: lib/resources/postgres.rb (hunk 1 above)
Comment (kidbrax) [reply lib/resources/postgres.rb:39]: I'm not sure.  Now that you mention it, I assume it is OS specific.  It is Alpine-based.
https://github.com/docker-library/postgres/blob/3d4e5e9f64124b72aa80f80e2635aff0545988c6/9.4/alpine/Dockerfile

Comment (aaronlippold) [reply lib/resources/postgres.rb:39]: @kidbrax @chris-rock @adamleff I folded this into PR #1758 so we can close both, it makes more sense to push the change there given the updates we made to the resource in general.

**Comment by adamleff** [comment]
Closing this issue, superseded by #1758

**Pull request closed without merging**